README.de.md

```
@@ -85,78 +88,101 @@ Unter [Releases](https://github.com/OpenCCU/OpenCCU/releases) steht für die jew
+   * [FAQ – Häufige Fragen](https://github.com/OpenCCU/OpenCCU/wiki/Support#faq)
    * [Fehler melden](https://github.com/OpenCCU/OpenCCU/wiki/Support#bugreports)
-   * [Features wünschen](https://github.com/OpenCCU/OpenCCU/wiki/Support#featurerequests)
-   * [Mitarbeit / Weiterentwicklung](https://github.com/OpenCCU/OpenCCU/wiki/Support#mitarbeit--weiterentwicklung)
+   * [Funktionen anfragen](https://github.com/OpenCCU/OpenCCU/wiki/Support#featurerequests)
+   * [Mitarbeit & Weiterentwicklung](https://github.com/OpenCCU/OpenCCU/wiki/Support#mitarbeit--weiterentwicklung)
 
-## :yum: Support, Mitarbeit
+## :yum: Support & Beiträge
 
-Für Feedback jeglicher Art, Fragen, Vorschläge, Wünsche und Fehlerberichte wird darum gebeten bitte zuerst im jeweiligen OpenCCU Teil des HomeMatic-Forum (<https://homematic-forum.de/forum/viewforum.php?f=65>) einen entsprechenden Beitrag zu erstellen oder an bereits existierenden Beiträgen dort sich zu beteiligen. Sollte aus den Diskussionen dort sich dann ein spezieller Feature-Wunsch oder von anderen Personen bestätigter Bug in OpenCCU herauskristallieren wird darum gebeten diesen Bug bzw. diesen Feature-Request im [Issue Tracker](https://github.com/OpenCCU/OpenCCU/issues) von OpenCCU entsprechend einzutragen bzw. zu melden.
+**Wo diskutieren / fragen**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)